src/pymor/parameters/base.py

```diff
@@ -374,6 +376,34 @@ def __eq__(self, mu):
                 return False
         return self.keys() == mu.keys() and all(np.array_equal(v, mu[k]) for k, v in self.items())
 
+    def __neg__(self):
+        return Mu({key: -value for key, value in self.items()})
+
+    def __add__(self, other):
+        if not isinstance(other, Mu):
+            try:
+                other = self.parameters.parse(other)
+            except Exception:
+                raise NotImplementedError
```

Comment:
I'm not sure what you are trying to achieve by catching exceptions here. `NotImplementedError` suggests that there could be an implementation, but I don't see how. Also, catching an arbitrary `Exception` is considered bad practice. Overall, I'd just go with:

```suggestion
            other = self.parameters.parse(other)
```